lib/action_subscriber.rb

```diff
@@ -80,6 +80,10 @@ def self.start_subscribers
     print_subscriptions
   end
 
+  def self.stop_subscribers!
+    @route_set.cancel_consumers! if @route_set
```

Comment:
I've pulled out the `route_set` to its own method that makes sure it is initialized to go along with your comments here. I think there is a future cut here to separate the list of routes from the state of  setting up consumers and declaring those queues. But for now this is a definite improvement :+1: